models/vrp.rb

```diff
@@ -268,6 +268,15 @@ def self.convert_position_relations(hash)
       relations_to_remove.reverse_each{ |index| hash[:relations].delete_at(index) }
     end
 
+    def self.deduce_first_solution_strategy(hash)
+      preprocessing = hash[:configuration] && hash[:configuration][:preprocessing]
+      return unless preprocessing
+
+      # To make sure that internal vrp conforms with the Grape vrp.
```

Comment:
We got bitten in the past because of this -- i.e., first_solution_strategy becoming array, string, nil, :horse: depending on the alignment of the stars -- it might have been because of tests.

But it is still useful because first_solution_strategy doesn't have `allow_blank: false` on the api side so it can be nil.